wremnants/pileup.py

```diff
@@ -22,17 +22,31 @@
             "2016H" : "H",
             "2016PreVFP" : "preVFP",
             "2016PostVFP" : "postVFP",
+            "2017" : "2017",
+            "2018" : "2018"
             }
 
 def make_pileup_helper(era = None, cropHighWeight = 5.,
                        filename_data = None,
-                       filename_mc = data_dir + "/pileupProfiles/MyMCPileupHistogram_2016Legacy_noGenWeights_preAndPostVFP.root"):
+                       filename_mc = None):
 
     # following the logic from https://github.com/WMass/cmgtools-lite/blob/7488bc844ee7e7babf8376d9c7b074442b8879f0/WMass/python/plotter/pileupStuff/makePUweightPerEra.py
 
     if filename_data is None:
-        filename_data = data_dir + f"/pileupProfiles/pileupProfileData_2016Legacy_Run{eradict[era]}_04June2021.root"
-
+        if era == "2016PostVFP":
```

Comment:
Trivial thing, to simplify this part of the code, you may define a dictionary keyed on the era, and having as value another dictionary with two elements, one for data and one for MC containing the files names. In this way the default definitions are all packed together tidily and then the "if era == ..." gets much simpler.